.github/workflows/linting.yml

```diff
@@ -13,4 +13,4 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/go-setup
-      - uses: golangci/golangci-lint-action@v5
+      - uses: golangci/golangci-lint-action@v4
```

Comment:
why did we downgrade from golangci-lint-action@v5 to golangci-lint-action@v4?

Reply:
That was an unnecessary change. It was intended to be an upgrade. 
The `golangci-lint-action` has been upgraded to v6 in [5f4986c](https://github.com/greenbone/opensight-golang-libraries/pull/62/commits/5f4986c2737100f35b29058feb35b588e701078b)

See release: https://github.com/golangci/golangci-lint-action/releases